lua/sqls/commands.lua

```
@@ -7,6 +7,9 @@ local M = {}
 
 local function make_show_results_handler(mods)
     return function(err, result, _, _)
+        if not result then
```

I think checking the value of `result` before `err` silences errors, could you move these checks below `if err`? Otherwise LGTM, thank you for the PR!

---

You are right, I'll change it.

---

Thanks again!